bin/zks

```diff
@@ -10,6 +19,6 @@ fzf --ansi --height 100% --preview 'zk-fts-search --backup -f {} {q} | bat --lan
     tmux send-keys -t \{left\} -l {} && \
     tmux send-keys -t \{left\} Enter \
   ]" \
-  --bind "change:reload:zk-fts-search --backup '{q}'" \
+  --bind "change:reload(sleep 0.01s; zk-fts-search --backup '{q}')" \
```

Comment:
This might have been a macos specific issue? but i got plenty of fzf update failures as it seemed to be _too fast_ and something (maybe the filesystem didn't  remove a lock file, or sqlite couldn't keep up?!) didn't like it, an slight artificial delay resolved the myriad of `index.db is locked by another process` type of errors (which seemed to be the previous fzf spawned process, nothing else was touching that file)